src/main/java/kitchenpos/table/application/TableService.java

```diff
@@ -14,11 +12,12 @@
 
 @Service
 public class TableService {
-    private final OrderRepository orderRepository;
+    private static final String NOT_EXIST_ORDER_TABLE_MESSAGE = "존재하지 않는 주문테이블 입니다.";
```

Comment:
행 구분이 있으면 좋을 것 같아서 체크해요 :)
```suggestion

    private static final String NOT_EXIST_ORDER_TABLE_MESSAGE = "존재하지 않는 주문테이블 입니다.";
    
```